.ci/e2e-expected/ddl-dml-batching.txt

```diff
@@ -1 +1 @@
-ERROR: FAILED_PRECONDITION: Executing updates is not allowed for DDL batches. "CREATE TABLE testtable (id integer PRIMARY KEY, data integer); INSERT INTO testtable (id, data) VALUES (1, 2); INSERT INTO testtable (id, data) VALUES (3, 4);"
+ERROR: FAILED_PRECONDITION: Executing updates is not allowed for DDL batches. "CREATE TABLE testtable (id integer PRIMARY KEY, data integer); INSERT INTO testtable (id, data) VALUES (1, 2); INSERT INTO testtable (id, data) VALUES (3, 4)"
```

Comment:
This PR removes the PGAdapter specific parser for removing comments and uses the one in the Java client library as well. There is one small difference between the two; the one in the Java client library will remove any trailing semicolon from the SQL string. This does not semantically change the commands, as a trailing semicolon in a SQL statement is ignored.